system/mount.py

```diff
@@ -348,6 +348,28 @@ def umount(module, dest):
     else:
         return rc, out+err
 
+def remount(module, mount_bin, **args):
+    ''' will try to use -o remount first and fallback to unmount/mount if unsupported'''
+    msg = ''
+    cmd = [mount_bin]
+    if get_platform().lower().endswith('bsd'):
+        cmd += ['-u']
+        if 'fstab' in args:
+            cmd += ['-F', args['fstab']]
+    else:
+        cmd += ['-o', 'remount' ]
+    cmd += [ args['name'], ]
+    try:
+        rc, out, err = module.run_command(cmd)
+    except:
+        rc = 1
+    if rc != 0:
+        msg = out+err
+        if os.path.ismount(args['name']):
```

Comment:
I'm not sure if we need this (we're in this function because we're supposed to remount an already mounted filesystem).

If we do need a conditional, we need to use the  ismount() function in this module.  (os.path.ismount() doesn't detect bind mounts).